utils/vifdump

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+##
+## vRouter/DPDK tcpdump
+##
+
+MON_IF_NAME="mon"
+VR_MAX_INTERFACES=63
+MON_TYPE="monitoring"
+
+usage () {
+    echo "vRouter/DPDK tcmpdump"
+    echo "Usage: ${0##*/} <vif index> [tcpdump arguments]"
+    echo "Example:"
+    echo "      ${0##*/} 1 -n"
+    exit 1
+}
+
+error () {
+    echo "${0##*/} error: $1"
+    exit 1
+}
+
+
+if [ $# -lt 1 ]; then
+    usage
+fi
+
+MONITORED_VIF_ID=${1}; shift
+
+## find free monitoring interface ID
+MON_IF_ID=0
+while true; do
+    if ! ifconfig ${MON_IF_NAME}${MON_IF_ID} &>/dev/null; then
```

Comment:
mon interface can use the vif index as the MON_IF_ID i.e. mon0 monitors vif0 and so on.